jest.config.js

```diff
@@ -1,17 +1,17 @@
 // jest.config.js
 module.exports = {
   verbose: true,
-  preset: "@testing-library/react-native",
+  preset: "react-native",
```

Comment:
hmmm could you share more about this change?

Reply:
this one is on the migration note.
https://callstack.github.io/react-native-testing-library/docs/migration-v7/#no-custom-jest-configuration